src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -146,6 +146,11 @@ class FluxPipeline(DiffusionPipeline, SD3LoraLoaderMixin):
     r"""
     The Flux pipeline for text-to-image generation.
 
+    Note:
+        This pipeline expects `latents` to be in a packed format. If you're providing
+        custom latents, make sure to use the `_pack_latents` method to prepare them.
+        Packed latents should be a 3D tensor of shape (batch_size, num_patches, channels).
```

Comment:
I think this note is unnecessary as well given that `_pack_latents()` is an inexpensive operation. I think your updates to `check_inputs()` should do the trick.

Reply:
Agreed, maybe then I can add this note as part of the `latents` docstring in the `__call__`